{"name": "test_dodawania[5-8]", "status": "failed", "statusDetails": {"message": "AssertionError: Suma dwóch takich samych składników powinna być równa 10\nassert (5 + 5) == 8", "trace": "skladnik = 5, suma = 8\n\n    @pytest.mark.parametrize(\"skladnik,suma\", [(5, 8), (2, 6)])\n    def test_dodawania(skladnik, suma):\n>       assert skladnik + skladnik == suma, \"Suma dwóch takich samych składników powinna być równa \" + str(skladnik + skladnik)\nE       AssertionError: Suma dwóch takich samych składników powinna być równa 10\nE       assert (5 + 5) == 8\n\npytest_project\\dodawanie_test.py:6: AssertionError"}, "parameters": [{"name": "skladnik", "value": "5"}, {"name": "suma", "value": "8"}], "start": 1669994097952, "stop": 1669994097952, "uuid": "f10900b0-67c3-4080-83a1-4a6a52ca9c75", "historyId": "fedd360367bb68a13adcc42c32d7720c", "testCaseId": "b49f26c57393c83da8f9d29d97e4174b", "fullName": "pytest_project.dodawanie_test#test_dodawania", "labels": [{"name": "parentSuite", "value": "pytest_project"}, {"name": "suite", "value": "dodawanie_test"}, {"name": "host", "value": "DESKTOP-OM28FAJ"}, {"name": "thread", "value": "7416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_project.dodawanie_test"}]}